core-web/libs/ui/src/lib/components/dot-content-thumbnail/dot-content-thumbnail.component.ts

```diff
@@ -0,0 +1,129 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+
+const ICON_MAP = {
+    html: 'pi-file',
+    pdf: 'pi-file-pdf',
+    image: 'pi-image',
+    video: 'pi-video',
+    msword: 'pi-file-word',
+    doc: 'pi-file-word',
+    docx: 'pi-file-word'
+};
+
+export enum CONTENT_THUMBNAIL_TYPE {
+    image = 'image',
+    video = 'video',
+    icon = 'icon'
+}
+
+@Component({
+    selector: 'dot-content-thumbnail',
+    standalone: true,
+    imports: [CommonModule],
+    templateUrl: './dot-content-thumbnail.component.html',
+    styleUrls: ['./dot-content-thumbnail.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotContentThumbnailComponent implements OnInit {
+    src: string;
+    thumbnailIcon: string;
+    thumbnailType: CONTENT_THUMBNAIL_TYPE;
+
+    @Input() tempUrl: string;
+    @Input() inode: string;
+    @Input() name: string;
+    @Input() contentType: string;
+    @Input() iconSize: string;
+    @Input() titleImage: string;
```

Comment:
When is a good idea change to an object? cc: @oidacra @KevinDavilaDotCMS @hmoreras @manuelrojas @zJaaal

Reply:
This should be an object, like `thumbnailOptions`

Reply:
Yep, use a typed object better.

Reply:
I'll do it!

Reply:
Done!